app/assets/stylesheets/sidebar_admin.css.scss

```diff
@@ -34,3 +34,7 @@
     display: inline-block;
   }
 }
+
+h3.sidebar_title {
```

Comment:
It fixed an issue from #1. I lost the invisibility cloak so I can't hide this change.